test/TestDNSSEC.js

```diff
@@ -110,6 +110,30 @@ async function verifyFailedSubmission(instance, data, sig, proof) {
 }
 
 contract('DNSSEC', function(accounts) {
+  before(async () => {
```

Comment:
What are these changes for ?

Reply:
The new test fails without them, depending on whether the other tests are run and how quickly or slowly they execute. It depends on the state of the inception value in the oracle.

Reply:
The existing tests are written sequentially, with each building on the last. It's not ideal, but it's the way Truffle works. Can you add the new test in the same fashion?

Reply:
Done, however I think isolating the tests from each other is a worthwhile improvement. Can I please move that commit to its own pull request?

Reply:
Sure, that seems reasonable.